14-EventToCommandBehavior/src/PrismSample/ViewModels/EventArgsExamplePageViewModel.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.ObjectModel;
+using PrismSample.Services;
+using PrismSample.Models;
+using Prism.Services;
+using Prism.Commands;
+using Prism.Mvvm;
+
+namespace PrismSample.ViewModels
+{
+    public class EventArgsExamplePageViewModel : BindableBase
+    {
+        private readonly IPageDialogService _pageDialogService;
+
+        public ObservableCollection<Developer> Developers { get; set; }
+
+        private DelegateCommand<Developer> _selectedDeveloperCommand;
+        public DelegateCommand<Developer> SelectedDeveloperCommand => _selectedDeveloperCommand ?? (_selectedDeveloperCommand = new DelegateCommand<Developer>((developer) => SelectedDeveloper(developer)));
```

Comment:
let's keep things consistent and clean... follow the same pattern for declaring your Commands as on the MainPageViewModel

Reply:
Done with the latest commit. :)